packages/cactus-plugin-ledger-connector-besu/src/main/typescript/plugin-ledger-connector-besu.ts

```diff
@@ -230,7 +232,26 @@ export class PluginLedgerConnectorBesu
   > {
     return ConsensusAlgorithmFamily.AUTHORITY;
   }
-
+  public async hasTransactionFinality(): Promise<boolean> {
+    const currentConsensusAlgorithmFamily = await this.getConsensusAlgorithmFamily();
+
+    const unrecognizedConsensusAlgorithmFamily =
+      !(Object.values(
```

Comment:
@jscode017 I would cache the result of 
```typescript
(Object.values(
      ConsensusAlgorithmFamiliesWithTxFinality,
    ) as string[]).includes(currentConsensusAlgorithmFamily.toString())
```

So that at the end it can be returned as-is without further computation needed.